keel-sql/src/main/resources/db/changelog/20201218-add-environment-verify-with.yml

```diff
@@ -0,0 +1,13 @@
+databaseChangeLog:
+- changeSet:
+    id: add-environment-verify-with
+    author: fletch
+    changes:
+    - addColumn:
+        tableName: environment
+        columns:
+        - column:
+            name: verify_with
```

Comment:
I was torn between sticking with a field -> column naming convention here, or just use `verifications` to follow the pattern of `contraints` and `notifications` (although those do match their field names). I guess it doesn't matter that much, never mind.

Reply:
+1 to `verifications`

Reply:
I was thinking we might want to unify everything into a single column so we don't fall into this trap again

Reply:
Renamed the column as suggested